controllers/k8ssandra/medusa_reconciler_test.go

```diff
@@ -89,12 +118,35 @@ func createMultiDcClusterWithMedusa(t *testing.T, ctx context.Context, f *framew
 	require.NoError(err, "failed to create K8ssandraCluster")
 	verifyReplicatedSecretReconciled(ctx, t, f, kc)
 
+	reconcileMedusaStandaloneDeployment(ctx, t, f, kc, "dc1", f.DataPlaneContexts[0])
 	t.Log("check that dc1 was created")
 	dc1Key := framework.ClusterKey{NamespacedName: types.NamespacedName{Namespace: namespace, Name: "dc1"}, K8sContext: f.DataPlaneContexts[0]}
 	require.Eventually(f.DatacenterExists(ctx, dc1Key), timeout, interval)
 
 	verifySecretAnnotationAdded(t, f, ctx, dc1Key, cassandraUserSecret)
 
+	t.Log("check that the standalone Medusa deployment was created in dc1")
+	medusaDeploymentKey1 := framework.ClusterKey{NamespacedName: types.NamespacedName{Namespace: namespace, Name: medusa.MedusaStandaloneDeploymentName("test", "dc1")}, K8sContext: f.DataPlaneContexts[0]}
+	medusaDeployment1 := &appsv1.Deployment{}
+	require.Eventually(func() bool {
+		if err := f.Get(ctx, medusaDeploymentKey1, medusaDeployment1); err != nil {
+			return false
+		}
+		return true
+	}, timeout, interval)
+
+	require.True(f.ContainerHasEnvVar(medusaDeployment1.Spec.Template.Spec.Containers[0], "MEDUSA_RESOLVE_IP_ADDRESSES", "False"))
```

Comment:
Question: what does this do?

Question2: can we test that the environment variable containing the mappings has been created?

Reply:
Ignore q2, I found you're testing this in the restore job controller test.

Reply:
So it turns off reverse DNS resolution by the sound of it. OK.